src/utils/data/db.js

```diff
@@ -1,15 +1,16 @@
 /**
  * Get connection with MongoDB
- * 
+ *
  * When need write route or get data functions, import "connect" from this file
```

Comment:
How about write an example code like "import connect from "./db";" in this description.

Reply:
OK